Changes made from .js to cdn

JStevens625 opened the pull request:
Bower components deleted, Index.html changed directory from bower to cdn
compenents

Commit:
Changes made from .js to cdn

Bower components deleted, Index.html changed directory from bower to cdn
compenents

dvigne commented on index.html:
```diff
@@ -113,15 +113,15 @@ <h2 class="support-text">Having Computer Troubles?</h1>
     <!-- Invlude Scrollreveal JS -->
-    <script type="text/javascript" src="bower_components\scrollreveal\dist\scrollreveal.min.js"></script>
+    <script <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/scrollReveal.js/3.3.1/scrollreveal.min.js"></script>
     <script type="text/javascript" src="assets\js\jquery-easing.js"></script>
```
Jacob, could you please add the CDN for jquery-easing.js? Thank you!

dvigne reviewed (changes requested):
Could you also remove bower.json from the project base? Thanks!

Commit:
jquery easing was changed to cdn

In addition to the original files changed to cdn and the bower
components files being removed the jquery easing was changed to a cdn as
well

Commit:
Bower.json removed

Commit:
Unecessary file from easing removed
